Mixmatch/streetview_dataset/README.md

```diff
@@ -0,0 +1,93 @@
+|  streetview_v3_64.1@250-200  |   250  |    0.00005    |     64     |  77.19%  |
+|  streetview_v3_64.1@250-200  |   250  |     0.0001    |     64     |  76.66%  |
+|  streetview_v3_64.1@250-200  |   250  |     0.0005    |     64     |  78.65%  |
+|  streetview_v3_64.1@250-200  |   250  |     0.001     |     64     |  74.80%  |
+|  streetview_v3_64.1@250-200  |   250  |     0.002     |     64     |  78.91%  |
+| streetview_v3_64.1@20000-200 |  20000 |     0.0001    |     64     |   70.03  |
+| streetview_v3_64.1@39000-200 |  39000 |     0.0001    |     64     |   79.97  |
+
+![Learning Rate to Accuracy](./IMG/streetview_v3_learning_rate_pr.png?raw=true)
+
+---
+#### **Model Layer Depth to Accuracy**
+The layer depth is the numbers of layers within the Mixmatch residual network.
+
+|           Dataset          | Learning Rate | Batch Size | Layers (Depth) | Training Speed | Accuracy |
+|:--------------------------:|:-------------:|:----------:|:--------------:|:--------------:|:--------:|
+| streetview_v3_64.1@250-200 |     0.0001    |     64     |       46       |      300%      |   76.62  |
+| streetview_v3_64.1@250-200 |     0.0001    |     64     |       78       |      180%      |   79.44  |
+| streetview_v3_64.1@250-200 |     0.0001    |     64     |       110      |      130%      |   79.84  |
+| streetview_v3_64.1@250-200 |     0.0001    |     64     |       142      |      100%      |   78.65  |
+| streetview_v3_64.1@250-200 |     0.0001    |     64     |       174      |       81%      |   80.37  |
+| streetview_v3_64.1@250-200 |     0.0001    |     64     |       206      |       69%      |   78.38  |
+
+![Model Layer Depth to Accuracy](./IMG/streetview_v3_depth_size_pr.png?raw=true)
+
+---
+#### **Image Size to Accuracy**
+All the previous results used a 64x64 size dataset. Here we test using 256x256 images.
+NOTE: Due to machine limitation, on 256x256 dataset the batch size is shrieked from 64 to 8. Thus, adding a control group on 64x64 with batch size 8 for comparison.
+
+|           Dataset           | Image Size | Labels | Learning Rate | Batch Size | Accuracy |
+|:---------------------------:|:----------:|:------:|:-------------:|:----------:|:--------:|
+| streetview_v3_256.1@250-200 |   256x256  |   250  |     0.0001    |      8     |   68.17  |
+|  streetview_v3_64.1@250-200 |    64x64   |   250  |     0.0001    |     64     |   76.66  |
+|  streetview_v3_64.1@250-200 |    64x64   |   250  |     0.0001    |      8     |   67.77  |
+
+
+### Streetview_v4:
+There are two ways to add the word embedding to the model, one way is to concatenate the vectors after the image pixels, the other way is to feed them into the model at the top fully connected layers.
+![Model Layer Depth to Accuracy](./IMG/streetview_v4_embedding.png?raw=true)
+
+| Label Size | Learning Rate | Batch Size | Accuracy w/o word embeddings | Accuracy w/ word embeddings A method | Accuracy w/ word embeddings B method |
```

Comment:
Explain which approach (A or B) does this table use.

Reply:
Done